scripts/authz_proposal_for_validator_ops.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -e
+
+if [[ -z "${GRANTER}" ]]; then
+    echo -n GRANTER:
+    read granter 
+    export GRANTER="${granter}"
+    echo
+    printf 'Please run "export GRANTER=%s" to avoid setting it in the future or add it to your bashrc file.'
+    echo 
+    echo
+else
+    echo "\$GRANTER is set to '$GRANTER'"
+fi
+
+if [[ -z "${CHAIN_ID}" ]]; then
+    echo -n CHAIN_ID:
+    read chain_id 
+    export CHAIN_ID="${chain_id}"
+    echo
+    printf 'Please run "export CHAIN_ID=%s" to avoid setting it in the future or add it to your bashrc file.'
+    echo 
+    echo
+else
+    echo "\$CHAIN_ID is set to '$CHAIN_ID'"
+fi
+
+printf 'Grantees (reading from grantee.txt):'
+echo
+for GRANTEE in $(cat grantee.txt); do
+    echo " - $GRANTEE"
+done
+
+echo
+
+for GRANTEE in $(cat grantee.txt); do
+    echo
+    echo "generating MsgSubmitProposal, MsgVote, MsgUnjail for grantee: '$GRANTEE'"
+    seid tx authz grant $GRANTEE generic --from admin -b block -y --fees 20000usei \
```

Comment:
change this to GRANTER

Reply:
specifically, --from will not always be from admin